src/Cameras/VR/babylon.webVRCamera.ts

```diff
@@ -175,26 +194,10 @@ module BABYLON {
         }
 
         public useStandingMatrix = (callback = (bool: boolean) => { }) => {
-            // Use standing matrix if availible
-            if (!navigator || !navigator.getVRDisplays) {
-                callback(false);
-            } else {
-                navigator.getVRDisplays().then((displays: any) => {
-                    if (!displays || !displays[0] || !displays[0].stageParameters || !displays[0].stageParameters.sittingToStandingTransform) {
-                        callback(false);
-                    } else {
-                        this._standingMatrix = new Matrix();
-                        Matrix.FromFloat32ArrayToRefScaled(displays[0].stageParameters.sittingToStandingTransform, 0, 1, this._standingMatrix);
-                        if (!this.getScene().useRightHandedSystem) {
-                            [2, 6, 8, 9, 14].forEach((num) => {
-                                if (this._standingMatrix) {
-                                    this._standingMatrix.m[num] *= -1;
-                                }
-                            });
-                        }
-                        callback(true)
-                    }
-                })
+            this._useStandingMatrix = true;
+            this._useStandingMatrixCallback = callback;
+            if (!this._vrDevice) {

```

Comment:
if this._vrDevice already exists, the callback should be executed and the standing matrix should be initialized (unless it is auto-initialized like in my first comment)